lib/Crypto/Util/asn1.py

```diff
@@ -486,7 +486,7 @@ def decode(self, der_encoded, nr_elements=None, only_ints_expected=False, strict
                     The number of members the SEQUENCE can have
                   only_ints_expected (boolean):
                     Whether the SEQUENCE is expected to contain only integers.
-                  struct (boolean):
```

Comment:
Shouldn't the argument order should be adjusted as well to reflect the changed signature?

Reply:
Not quite sure what you mean.

Reply:
Usually, the documentation order of arguments match the order of arguments in the signature...